kentik_api_library/tests/component/test_alerting.py

```diff
@@ -0,0 +1,34 @@
+from http import HTTPStatus
+
+from kentik_api.api_resources.alerting_api import AlertingAPI
+from kentik_api.api_calls.api_call import APICallMethods
+from kentik_api.public.manual_mitigation import ManualMitigation
+from tests.component.stub_api_connector import StubAPIConnector
+
+
+def test_create_manual_mitigation_success() -> None:
+    # given
+    create_response_payload = """
+    {
+        "response": {
+            "result": "OK"
+        }
+    }"""
+    connector = StubAPIConnector(create_response_payload, HTTPStatus.OK)
+    alerting_api = AlertingAPI(connector)
+
+    # when
+    manual_mitigation = ManualMitigation("192.168.1.0/24", "1234", "12345", "20", "comment")
+    created = alerting_api.create_manual_mitigation(manual_mitigation)
+
+    # then
+    assert connector.last_url == "/alerts/manual-mitigate"
```

Comment:
```suggestion
    assert connector.last_url_path == "/alerts/manual-mitigate"
```
I think that connector's field should be renamed to be precise

Reply:
This little change will cost us many files modifications, because this connector is used in many more component tests. I think this suggestion is accurate but this will be small improvement compared to extra work it needs.

Reply:
You are right, I changed field name to `last_url_path`